commands/build.go

```diff
@@ -835,7 +835,7 @@ func printWarnings(w io.Writer, warnings []client.VertexWarning, mode progressui
 		fmt.Fprintf(sb, "%d warnings found", len(warnings))
 	}
 	if logrus.GetLevel() < logrus.DebugLevel {
-		fmt.Fprintf(sb, " (use --debug to expand)")
```

Comment:
It looks like you want to debug an issue. Could you please provide more details about what you need help with? For example:
- The specific problem or error you're encountering
- The repository or code file in question
- Any steps you've already taken to debug the issue

This information will help me assist you more effectively.